src/mongo/base/CompositeBase.class.php

```diff
@@ -12,111 +14,105 @@
      * @var \Tripod\Mongo\Jobs\ApplyOperation
      */
     protected $applyOperation;
+
+    protected $batchSize = 1;
```

Comment:
Is this used? I can see a `$batchSize` in config - and when you get the batch size you are getting it from the config object. I can't see this being used at all.